bgpd/bgp_vty.c

```diff
@@ -9648,15 +9628,35 @@ static void bgp_show_neighbor_graceful_restart_capability_per_afi_safi(
 						"endOfRibSend");
 					json_object_boolean_false_add(
 						json_endofrib_status,
-						"endOfRibSentAfterUpdate");
+						"endOfRibSendAfterUpdate");
 				} else {
 					vty_out(vty, "No\n");
 					vty_out(vty,
-						"     EoRSentAfterUpdate   : ");
+						"      End-of-RIB send after update: ");
```

Comment:
I see that the general notation in our CLI is to use start every word with Capital letter. Any specific reason to change this ?

Reply:
Actually we have mixed notations.. We should make them unified as well.

Reply:
Yes. we should unify. +1 for that.

My earlier comment may be little confusing. Changing EorSentAfterUpdate is the right thing to do.  I meant ""End-of-RIB Sent After ... "" suits mostly with the existing format.